dgbpy/dgbcombine.py

```diff
@@ -0,0 +1,131 @@
+modeltypes = ['classification', 'regression']
+defsavetype = savetypes[2]
+defmodeltype = modeltypes[0]
+
+def fit(models, inputs, outputs): # models, Xs, ys
+    '''
+    Train and combine multiple models with the dataset (input and output) per each model.
+    
+    models = list of models
+    inputs = list of inputs
+    outputs = list of outputs
+    
+    example
+    -----------------------------
+    models = [Model1(), Model2(), Model3(probability=True)]
+    inputs = [X1, X2, X3]
+    outputs = [y1, y2, y3]
+    -----------------------------
+    
+    Model1 will be train based on X1 and y1
+    Model2 will be train based on X2 and y2
+    So on....
+    
+    # Return
+    Trained/fitted list of models
+    '''
+    dictModel = {'model':models,
+                 'input':inputs,
+                 'output':outputs,
+                 'trained':[]}
+    for i in range(0, len(dictModel['model'])):
+        X_train_, y_train_ = dictModel['input'][i], dictModel['output'][i]
+        dictModel['trained'].append(dictModel['model'][i].fit(X_train_, y_train_))
+    return dictModel['trained']
+    
+def predict(models, X, model_type=defmodeltype): 
+    '''
+    Predict the data with trained multiple models.
+    
+    models = list of trained models
+    X = Inputs
+    model_type = classification or regression
+    '''
+    if model_type == modeltypes[0]:
+        return np.argmax((np.sum([models[i].predict_proba(X) for i in range(0, len(models))], axis=0)), axis=1)
+    elif model_type == modeltypes[1]:
+        return np.mean(np.column_stack([model.predict(X) for model in models]), axis=1)
+    
+def predict_onnx(sess, X, model_type=defmodeltype):
+    '''
+    Predict the data with trained multiple models that saved using onnx format.
+    
+    sess = onnx's session
+    X = Inputs
+    model_type = classification or regression
+    '''
+    if model_type == modeltypes[0]:
+        input_names = [i.name for i in sess.get_inputs()]
+        output_names = [o.name for o in sess.get_outputs()]
+        input_feed = {}
+        for i in input_names:
+            input_feed[i] = X.astype(np.float32)
+        outputs = sess.run(output_names, input_feed)
+        prob = []
+        for i in range(len(outputs)):
+            if i % 2 == 1:
+                prob_ = []
+                for out in outputs[i]:
+                    prob_.append(list(out.values()))
+                prob.append(prob_)
+        return np.argmax((np.sum(prob, axis=0)), axis=1)
+    elif model_type == modeltypes[1]:
+        input_names = [i.name for i in sess.get_inputs()]
+        output_names = [o.name for o in sess.get_outputs()]
+        input_feed = {}
+        for i in input_names:
+            input_feed[i] = X.astype(np.float32)
+        outputs = sess.run(output_names, input_feed)
+        return np.mean(np.column_stack(outputs), axis=1)   
+
+def save_model(models, outfnm, save_type=defsavetype, input_shape=None):
```

Comment:
Mostly duplicates dgbscikit.py:save
Let me propose the following signature:
dgbscikit.py:save( model, outfnm, save_type=defsavetype, prior_models=None )

Where prior_models is a set of already trained models.
If provided, you can add 'model' to the list 'prior_models', and save that extended list